custom_components/tuya_local/devices/9in1_airquality_monitor.yaml

```diff
@@ -0,0 +1,161 @@
+  dps:
+    - id: 1
+      type: string
+      name: sensor
+      readonly: true
+      mapping:
+        - dps_val: "level_1"
+          value: "Excellent"
+        - dps_val: "level_2"
+          value: "Good"
+        - dps_val: level_3
+          value: "Lightly polluted"
+
+secondary_entities:
+  - entity: sensor
+    name: Temperature
+    class: temperature
+    dps:
+      - id: 2
+        type: integer
+        name: sensor
+        unit: C
+        class: measurement
+  - entity: sensor
+    name: Humidity
+    class: humidity
+    icon: "mdi:water-percent"
+    dps:
+      - id: 3
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+        range:
+          min: 0
+          max: 100
+  - entity: sensor
+    name: CO2
+    class: carbon_dioxide
+    icon: "mdi:factory"
+    dps:
+      - id: 4
+        type: integer
+        name: sensor
+        unit: ppm
+        class: measurement
+  - entity: sensor
+    name: CO
+    class: carbon_monoxide
+    icon: "mdi:factory"
+    dps:
+      - id: 102
+        type: integer
+        name: sensor
+        unit: ppm
+        class: measurement
+  - entity: sensor
+    name: HCHO
+    class: volatile_organic_compounds
+    icon: "mdi:factory"
+    dps:
+      - id: 5
+        type: integer
+        name: sensor
+        unit: µg/m³
+        class: measurement
+        range:
+          min: 0
+          max: 1000
+        mapping:
+          - scale: 1
+            step: 1
+  - entity: sensor
+    name: TVOC
+    class: volatile_organic_compounds
+    icon: "mdi:factory"
+    dps:
+      - id: 6
+        type: integer
+        name: sensor
+        unit: µg/m³
+        class: measurement
+        range:
+          min: 0
+          max: 999
+        mapping:
+          - scale: 1
+            step: 1
+  - entity: sensor
+    name: PM 2.5
+    class: pm25
+    icon: "mdi:factory"
+    dps:
+      - id: 7
+        type: integer
+        name: sensor
+        unit: µg/m³
+        class: measurement
+        range:
+          min: 0
+          max: 999
+  - entity: sensor
+    name: PM 10
+    class: pm10
+    icon: "mdi:factory"
+    dps:
+      - id: 9
+        type: integer
+        name: sensor
+        unit: µg/m³
+        class: measurement
+        range:
+          min: 0
+          max: 999
+  - entity: sensor
+    name: Battery
+    class: battery
+    category: diagnostic
+    dps:
+      - id: 22
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+        range:
+          min: 0
+          max: 100
+  - entity: sensor
+    name: Charge State
+    class: enum
+    category: diagnostic
+    dps:
+      - id: 23
+        type: boolean
+        name: sensor
+        mapping:
+          - dps_val: false
+            value: "Battery"
+            icon: "mdi:battery"
+          - dps_val: true
+            value: "Charging"
+            icon: "mdi:battery-charging"
+            default: true
```

Comment:
If the intention is to make this apply in the case where the dp is missing, or has an unknown value, then removing `dps_val: true` is the way to do that. 

The `default` attribute is used only in one specific use case for siren tones, where it is not intended to appear as the current value, but when activating the siren, a tone is needed even if none is being returned as the .current value.